plugins/woocommerce/includes/admin/settings/class-wc-settings-emails.php

```diff
@@ -277,6 +277,8 @@ protected function get_settings_for_default_section() {
 		$body_text_color_setting_in_palette   = $reorder_colors ? $body_text_color_setting : null;
 		$footer_text_color_setting_in_palette = $reorder_colors ? $footer_text_color_setting : null;
 
+		$block_email_editor_enabled = get_option( 'woocommerce_feature_block_email_editor_enabled', 'no' );
```

Comment:
I think we should rather use  `FeaturesUtil::feature_is_enabled( 'block_email_editor' )` for this check, and we get `true` or `false`. I think it is better than checking the `no` string as it may theoretically be '0' or something else.

Reply:
Thanks for this. I had it in my mind, but in the end, I forgot.